### PR TITLE
kernel: add ppc64le fragments

### PR DESCRIPTION
**Pull request by Amulyam24**
Adding support for ppc64le kernel fragments.

Fixes: #1898

Signed-off-by: Amulyam24 <amulmek1@in.ibm.com>

### CODE REVIEW

File: tools/packaging/kernel/configs/fragments/arm64/base.conf [hunk 1]
```diff
@@ -42,3 +42,4 @@ CONFIG_ARM64_UAO=y
 
 CONFIG_NO_HZ_FULL=y
 CONFIG_GENERIC_MSI_IRQ_DOMAIN=y
+CONFIG_RANDOMIZE_BASE=y
```
Comment (devimc) [inline tools/packaging/kernel/configs/fragments/arm64/base.conf:45]: *spidey sense:* this may impact the performance, let's run the metrics CI

Comment (clnperez) [reply tools/packaging/kernel/configs/fragments/arm64/base.conf:45]: @devimc I talked to @Amulyam24 about this one as well. It was in the base before, so this actually is only removing it for ppc64le. It's a 32-bit only option for Power atm, but seems to be fine to use by default on all other platforms (unless we're misunderstanding what the base conf file means of course).

Comment (devimc) [reply tools/packaging/kernel/configs/fragments/arm64/base.conf:45]: @clnperez metrics CI looks good

**Comment by clnperez** [comment]
okay to merge this one? or do we need to re-run `jenkins-ci-ARM-ubuntu-18-04` ?

**Comment by fidencio** [comment]
> okay to merge this one? or do we need to re-run `jenkins-ci-ARM-ubuntu-18-04` ?

We don't necessarily needed it, but I'd like to rerun it.

/test-arm

**Review by fidencio — APPROVED** [review]
lgtm, thanks @Amulyam24!

**Review by bergwolf — APPROVED** [review]

**Pull request merged**